.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.21-openshift-4.16
+  tag: rhel-9-release-golang-1.21-openshift-4.16
```

Comment:
If this is backported, this should probably be a 4.14 image ref?

Reply:
This is the same issue as 4.15, there is no go lang 1.21 image prior to 4.16

Reply:
ok